packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/__snapshots__/ColumnChooser.component.test.js.snap

```diff
@@ -1,6 +1,30 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ColumnChooser should render with default props 1`] = `
+<form class="theme-filter tc-column-chooser-filter theme-tc-column-chooser-filter theme-navbar"
+      role="search"
+>
+  <div class="form-group">
+    <svg name="talend-search"
+         class="theme-tc-svg-icon tc-svg-icon theme-search-icon"
+         focusable="false"
+         aria-hidden="true"
+    >
+      <use xlink:href="#talend-search">
+      </use>
+    </svg>
+    <input name="search"
+           placeholder="Find a column"
+           autocomplete="off"
+           aria-label="Find a column"
+           role="searchbox"
+           type="search"
+           id="my-id-filter-input"
+           class="theme-search form-control"
+           value
+    >
+  </div>
+</form>
```

Comment:
This is a weird change. This snapshot is correct, looking at the code, we have a Fragment that contains the search form (everytime) before the column chooser. And it hasn't changed since the file creation.

My guess: an old bug in jest snapshot and react fragment that doesn't print all the element ¯\_(ツ)_/¯